packages/odyssey-react-mui/src/labs/Switch.tsx

```diff
@@ -341,21 +369,27 @@ const Switch = ({
             odysseyDesignTokens={odysseyDesignTokens}
             type="checkbox"
             value={value}
+            tabIndex={isReadOnly ? 0 : undefined}
```

Comment:
`0` is the default value. Feels like this isn't really doing anything. But, maybe I'm missing something

Reply:
Good catch, this was an artifact from something I was exploring, I just verified this is not needed.